lib/roda/plugins.rb

```diff
@@ -6,6 +6,7 @@ class Roda
   # Module in which all Roda plugins should be stored. Also contains logic for
   # registering and loading plugins.
   module RodaPlugins
+    NOOP = ->(*args) { args }.freeze
```

Comment:
I opted for this because you seem to prefer to reduce the number of allocations and this is a general function. Happy to remove it if you'd rather see it inlined.

Reply:
`NOOP` is probably the wrong name as I'm reading this again; it's an identity function, not a no-op.